unstable test `TestTCPServerInsecureHTTP1 `

```
=== Failed
=== FAIL: pkg/tcpserver TestTCPServerInsecureHTTP1 (0.00s)
[2025/11/11 07:14:50.106 +08:00] [INFO] [tcp_server.go:113] ["tcp server start to serve"]
[2025/11/11 07:14:50.107 +08:00] [ERROR] [tcp_server.go:131] ["tcp server error"] [error="accept tcp 127.0.0.1:40139: use of closed network connection"] [stack="github.com/pingcap/ticdc/pkg/tcpserver.(*tcpServerImpl).Run.func3\n\t/home/prow/go/src/github.com/pingcap/ticdc/pkg/tcpserver/tcp_server.go:131"]
[2025/11/11 07:14:50.107 +08:00] [INFO] [tcp_server.go:115] ["tcp server exited"]
    tcp_server_test.go:58: 
        	Error Trace:	/home/prow/go/src/github.com/pingcap/ticdc/pkg/tcpserver/tcp_server_test.go:58
        	            				/home/prow/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.7.linux-amd64/src/runtime/asm_amd64.s:1700
        	Error:      	Expect "context canceled" to match ".*ErrTCPServerClosed.*"
        	Test:       	TestTCPServerInsecureHTTP1
```

--- a/pkg/tcpserver/tcp_server.go
+++ b/pkg/tcpserver/tcp_server.go
@@ -27,6 +27,7 @@ import (
 	"github.com/soheilhy/cmux"
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
 )
 
 var cmuxReadTimeout = 10 * time.Second

